assets/src/edit-story/elements/image/layer.js

```diff
@@ -13,25 +13,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * External dependencies
+ */
+import { __ } from '@web-stories-wp/i18n';
 
 /**
  * Internal dependencies
  */
 import StoryPropTypes from '../../types';
-import VisibleImage from '../media/visibleImage';
-import { getSmallestUrlForWidth } from '../media/util';
+import { LayerText, LayerTextPropTypes } from '../shared/layerText';
 
-function ImageLayerContent({
-  element: {
-    resource,
-    resource: { alt },
-  },
-}) {
-  const src = getSmallestUrlForWidth(0, resource);
-  return <VisibleImage src={src} alt={alt} height="20" />;
-}
+function ImageLayerContent({ isBackground, element }) {
+  const { alt } = element?.resource || {};
 
+  return (
+    <LayerText isBackground={isBackground}>
+      {alt || __('Image', 'web-stories')}
```

Comment:
Should this be `Image Content`?
```suggestion
      {alt || __('Image Content', 'web-stories')}
```